LICENSES/BSD-3-Clause.txt

```diff
@@ -0,0 +1,11 @@
+Copyright (c) <year> <owner>. 
```

Comment:
The point (as I understand the REUSE spec and issues/discussions) is to provide _unaltered_ versions of the licenses. A handful of open source licenses (MIT, BSD-3-Clause, etc.) include copyright claims within the license file while most others (Apache-2.0, GPL-3.0, etc.) do not.

The application of the copyright and licenses are then done elsewhere via specific file annotations: in file, in a matching `.license` file, or via the `.reuse/dep5` annotations.

Reply:
I do think word-wrapping would be ok.  I'm not sure there's an official source of the license, and I think it's almost universally word wrapped when presented in plain text.

Reply:
I'm leaving this as is because modifying the output from `reuse download` would just be annoying additional effort. Also, just word wrapping it would make it a "third thing" which is not either the officially formatted one from the ASF nor the completely unwrapped one from REUSE. So...gonna leave well enough alone on this one.